_content/tour/functions.article

```diff
@@ -6,10 +6,7 @@ Functions are at the core of the language and they provide a mechanism to group
 - [[https://www.ardanlabs.com/training/individual-on-demand/ultimate-go-bundle/][Watch The Video]]
 - Need Financial Assistance, Use Our [[https://www.ardanlabs.com/scholarship/][Scholarship Form]]
 
-Functions are at the core of the language and they provide a mechanism to group
-and organize our code to separate and distinct pieces of functionality. They can
-be used to provide an API to the packages we write and are a core component
-to concurrency.
+Functions are at the core of the language and they provide a mechanism to group and organize our code to separate and distinct pieces of functionality. They can be used to provide an API to the packages we write and are a core component to concurrency.
```

Comment:
This the sentence doesn't start with a `-`, then please line break at or near column 80.

Reply:
I don't think we want this change unless you are fixing a typo